bug: TestExecOutput Fails

## Describe the bug
TestExecOutputs is calling to a hardcoded URL that is 404-ing, because it has been moved to [here](https://github.com/jqlang/jq/tree/master/sig) - we should update all references appropriately 


```
     outputs_test.go:91: 
2023-08-01T18:15:35.9057166Z         	Error Trace:	/home/vsts/work/1/s/tests/integration/outputs_test.go:91
2023-08-01T18:15:35.9057387Z         	            				/home/vsts/work/1/s/tests/integration/outputs_test.go:25
2023-08-01T18:15:35.9057592Z         	Error:      	Received unexpected error:
2023-08-01T18:15:35.9058961Z         	            	unable to build CNAB invocation image: error building docker image: failed to solve: executor failed running [/bin/sh -c wget --no-check-certificate https://raw.githubusercontent.com/stedolan/jq/master/sig/jq-release.key -O /tmp/jq-release.key &&     wget --no-check-certificate https://raw.githubusercontent.com/stedolan/jq/master/sig/v${JQ_VERSION}/jq-linux64.asc -O /tmp/jq-linux64.asc &&     wget --no-check-certificate https://github.com/stedolan/jq/releases/download/jq-${JQ_VERSION}/jq-linux64 -O /tmp/jq-linux64 &&     gpg --import /tmp/jq-release.key &&     gpg --verify /tmp/jq-linux64.asc /tmp/jq-linux64 &&     cp /tmp/jq-linux64 /usr/bin/jq &&     chmod +x /usr/bin/jq &&     rm -f /tmp/jq-release.key &&     rm -f /tmp/jq-linux64.asc &&     rm -f /tmp/jq-linux64]: exit code: 8
2023-08-01T18:15:35.9059701Z         	Test:       	TestExecOutputs
2023-08-01T18:15:35.9059916Z --- FAIL: TestExecOutputs (21.66s)

```

--- a/tests/integration/build_test.go
+++ b/tests/integration/build_test.go
@@ -124,7 +124,7 @@ func TestRebuild(t *testing.T) {
 	// Explain the bundle a bunch more times, it should not rebuild again.
 	// This is a regression test for a bug where the manifest would be considered out-of-date when nothing had changed
 	// caused by us using a go map when comparing the mixins used in the bundle, which has inconsistent sort order...
-	for i := 0; i < 10; i++ {
+	for i := 0; i < 5; i++ {
 		_, output = test.RequirePorter("explain")
 		tests.RequireOutputContains(t, output, "Bundle is up-to-date!", "expected the previous build to be reused")
 	}

--- a/tests/integration/testdata/bundles/exec-outputs/Dockerfile.tmpl
+++ b/tests/integration/testdata/bundles/exec-outputs/Dockerfile.tmpl
@@ -7,9 +7,9 @@ ARG BUNDLE_DIR
 RUN apt-get update && apt-get install -y wget && rm -rf /var/lib/apt/lists/*
 RUN apt-get update && apt-get install gnupg2 -y
 ENV JQ_VERSION='1.6'
-RUN wget --no-check-certificate https://raw.githubusercontent.com/jglang/jq/master/sig/jq-release-new.key -O /tmp/jq-release.key && \
-    wget --no-check-certificate https://raw.githubusercontent.com/jglang/jq/master/sig/v${JQ_VERSION}/jq-linux64.asc -O /tmp/jq-linux64.asc && \
-    wget --no-check-certificate https://github.com/jglang/jq/releases/download/jq-${JQ_VERSION}/jq-linux64 -O /tmp/jq-linux64 && \
+RUN wget --no-check-certificate https://raw.githubusercontent.com/jqlang/jq/master/sig/jq-release-new.key -O /tmp/jq-release.key && \
+    wget --no-check-certificate https://raw.githubusercontent.com/jqlang/jq/master/sig/v${JQ_VERSION}/jq-linux64.asc -O /tmp/jq-linux64.asc && \
+    wget --no-check-certificate https://github.com/jqlang/jq/releases/download/jq-${JQ_VERSION}/jq-linux64 -O /tmp/jq-linux64 && \
     gpg --import /tmp/jq-release-new.key && \
     gpg --verify /tmp/jq-linux64.asc /tmp/jq-linux64 && \
     cp /tmp/jq-linux64 /usr/bin/jq && \

--- a/tests/integration/testdata/bundles/exec-outputs/Dockerfile.tmpl
+++ b/tests/integration/testdata/bundles/exec-outputs/Dockerfile.tmpl
@@ -7,10 +7,10 @@ ARG BUNDLE_DIR
 RUN apt-get update && apt-get install -y wget && rm -rf /var/lib/apt/lists/*
 RUN apt-get update && apt-get install gnupg2 -y
 ENV JQ_VERSION='1.6'
-RUN wget --no-check-certificate https://raw.githubusercontent.com/stedolan/jq/master/sig/jq-release.key -O /tmp/jq-release.key && \
-    wget --no-check-certificate https://raw.githubusercontent.com/stedolan/jq/master/sig/v${JQ_VERSION}/jq-linux64.asc -O /tmp/jq-linux64.asc && \
-    wget --no-check-certificate https://github.com/stedolan/jq/releases/download/jq-${JQ_VERSION}/jq-linux64 -O /tmp/jq-linux64 && \
-    gpg --import /tmp/jq-release.key && \
+RUN wget --no-check-certificate https://raw.githubusercontent.com/jglang/jq/master/sig/jq-release-new.key -O /tmp/jq-release.key && \
+    wget --no-check-certificate https://raw.githubusercontent.com/jglang/jq/master/sig/v${JQ_VERSION}/jq-linux64.asc -O /tmp/jq-linux64.asc && \
+    wget --no-check-certificate https://github.com/jglang/jq/releases/download/jq-${JQ_VERSION}/jq-linux64 -O /tmp/jq-linux64 && \
+    gpg --import /tmp/jq-release-new.key && \
     gpg --verify /tmp/jq-linux64.asc /tmp/jq-linux64 && \
     cp /tmp/jq-linux64 /usr/bin/jq && \
     chmod +x /usr/bin/jq && \

--- a/tests/integration/testdata/bundles/exec-outputs/Dockerfile.tmpl
+++ b/tests/integration/testdata/bundles/exec-outputs/Dockerfile.tmpl
@@ -7,10 +7,11 @@ ARG BUNDLE_DIR
 RUN apt-get update && apt-get install -y wget && rm -rf /var/lib/apt/lists/*
 RUN apt-get update && apt-get install gnupg2 -y
 ENV JQ_VERSION='1.6'
-RUN wget --no-check-certificate https://raw.githubusercontent.com/jqlang/jq/master/sig/jq-release-new.key -O /tmp/jq-release.key && \
-    wget --no-check-certificate https://github.com/jqlang/jq/blob/master/sig/v${JQ_VERSION}/jq-linux64.asc -O /tmp/jq-linux64.asc && \
+
+RUN wget --no-check-certificate https://raw.githubusercontent.com/jqlang/jq/master/sig/jq-release-old.key -O /tmp/jq-release.key && \
+    wget --no-check-certificate https://raw.githubusercontent.com/jqlang/jq/master/sig/v${JQ_VERSION}/jq-linux64.asc -O /tmp/jq-linux64.asc && \
     wget --no-check-certificate https://github.com/jqlang/jq/releases/download/jq-${JQ_VERSION}/jq-linux64 -O /tmp/jq-linux64 && \
-    gpg --import /tmp/jq-release-new.key && \
+    gpg --import /tmp/jq-release.key && \
     gpg --verify /tmp/jq-linux64.asc /tmp/jq-linux64 && \
     cp /tmp/jq-linux64 /usr/bin/jq && \
     chmod +x /usr/bin/jq && \

--- a/tests/integration/testdata/bundles/exec-outputs/Dockerfile.tmpl
+++ b/tests/integration/testdata/bundles/exec-outputs/Dockerfile.tmpl
@@ -8,7 +8,7 @@ RUN apt-get update && apt-get install -y wget && rm -rf /var/lib/apt/lists/*
 RUN apt-get update && apt-get install gnupg2 -y
 ENV JQ_VERSION='1.6'
 RUN wget --no-check-certificate https://raw.githubusercontent.com/jqlang/jq/master/sig/jq-release-new.key -O /tmp/jq-release.key && \
-    wget --no-check-certificate https://raw.githubusercontent.com/jqlang/jq/master/sig/v${JQ_VERSION}/jq-linux64.asc -O /tmp/jq-linux64.asc && \
+    wget --no-check-certificate https://github.com/jqlang/jq/blob/master/sig/v${JQ_VERSION}/jq-linux64.asc -O /tmp/jq-linux64.asc && \
     wget --no-check-certificate https://github.com/jqlang/jq/releases/download/jq-${JQ_VERSION}/jq-linux64 -O /tmp/jq-linux64 && \
     gpg --import /tmp/jq-release-new.key && \
     gpg --verify /tmp/jq-linux64.asc /tmp/jq-linux64 && \

